src/audio/module_adapter/module_adapter.c

```diff
@@ -733,3 +733,87 @@ void module_adapter_free(struct comp_dev *dev)
 	rfree(mod);
 	rfree(dev);
 }
+
```

Comment:
We could use here a better explanation about why do we need set_large_config and get_..

Reply:
fixed.